Client/Client.py

```diff
@@ -284,12 +299,20 @@ def getCutechessCommand(arguments, data, nps):
         '{0}-{1}'.format(data['test']['engine'], data['test']['dev']['name'])
     )
 
+    # In case devnetwork is specified append EvalFile option
+    if devnetwork:
+        devflags = devflags + " option.EvalFile=" + devnetwork
+
     # Options for the Base Engine
     baseflags = '-engine dir=Engines/ cmd=./{0} proto={1} tc={2}{3} name={4}'.format(
         baseCommand, data['test']['base']['protocol'], timecontrol, baseoptions,
         '{0}-{1}'.format(data['test']['engine'], data['test']['base']['name'])
     )
 
+    # In case basenetwork is specified append EvalFile option
```

Comment:
Change the comment to

``# Add evaluation weights for the Base engine if specified ``

Reply:
Fixed.